stl/inc/ranges

```diff
@@ -2672,8 +2679,20 @@ namespace ranges {
             }
         };
 
+        template <class _BaseTy>
+        class _Inner_iter_base {};
+        template <forward_range _BaseTy>
+        class _Inner_iter_base<_BaseTy> {

```

Comment:
Nitpick: I would prefer to see a newline between the primary template and the specialization here.